app/views/organizations/adopter_fosterer/liked_pets/create.turbo_stream.erb

```diff
@@ -0,0 +1,7 @@
+<%= turbo_stream.replace "like_button_pet_#{@pet.id}" do %>
+  <div id="like_button_pet_<%= @pet.id %>" class="d-flex flex-column align-items-center">
+    <%= render partial: 'partials/like_button', locals: { pet: @pet, liked: true } %>
```

Comment:
```suggestion
    <%= render partial: 'partials/like_button', locals: { pet: @pet, liked:  @liked_pet } %>
```

Reply:
Yes this will be better @jmilljr24, I will make changes.